_episodes/01-short-introduction-to-Python.md

```diff
@@ -11,7 +11,10 @@ objectives:
     - "Perform mathematical operations in Python using basic operators."
     - "Define the following as it relates to Python: lists, tuples, and dictionaries."
 keypoints:
-    - "FIXME"
+    - "Python is an interpreted language which can be used interactively (executing one command at a time) or in scripting mode (executing a series of commands saved in file)."
```

Comment:
This keypoint mixes Python (capital P) programming language and `python` (lowercase p, frequently referred to as CPython) program used to execute scripts/programs written in Python programming language. I suggest breaking this into two (or more) keypoints:

> - Python is an interpreted programming language [with ... | that enables ... | ...]
> - `python` program can be used for interactive and iterative development of Python programs  using its interactive prompt or for executing Python programs from the command line.